internal/table/table.go

```diff
@@ -0,0 +1,33 @@
+package table
+
+import (
+	"reflect"
+
+	"github.com/ActiveState/cli/internal/output"
+)
+
+type Table struct {
+	Info        string      `opts:"hideKey"`
+	Rows        interface{} `opts:"hideKey"`
+	emptyOutput string
+}
+
+func (t *Table) MarshalOutput(format output.Format) interface{} {
+	if format == output.PlainFormatName {
+		value := reflect.ValueOf(t.Rows)
```

Comment:
I would rather not use `reflect` here. Alternatively we could not handle the empty output.